src/test/resources/files/Rule/Controversial/CamelCasePropertyName/testRuleDoesApplyForPropertyNameWithCapital.php

```diff
@@ -15,7 +15,7 @@
  * @link http://phpmd.org/
  */
 
-class testRuleDoesNotApplyForPropertyNameWithCapital
+class testRuleDoesApplyForPropertyNameWithCapital
 {
     public $AlsoValidPropertyName;
```

Comment:
```suggestion
    public $NotValidPropertyName;
```
Would be less missleading.

Reply:
Fixed.